maintenance.py

```diff
@@ -53,10 +68,18 @@ def wait_for_maintenance(callback):
     last_etag = '0'
 
     while True:
-        r = requests.get(
-            url,
-            params={'last_etag': last_etag, 'wait_for_change': True},
-            headers=METADATA_HEADERS)
+        logger.debug("start loop")
```

Comment:
this doesn't seem very helpful. a repeating "start loop" for each iteration.

Reply:
the loop runs for an hour at a time (so this message printed every hour if still running), so this helped us understand when this is still running and when this failed.